matter_server/client/client.py

```diff
@@ -231,6 +233,19 @@ async def send_device_command(
             interaction_timeout_ms=interaction_timeout_ms,
         )
 
+    async def read_attribute(
+        self,
+        node_id: int,
+        attribute_path: str | list[str],
```

Comment:
Should it be a list or just a single path?

Reply:
the same typo again, adjusted it